dapp/src/components/Header/ConnectWallet.tsx

```diff
@@ -111,22 +115,24 @@ export default function ConnectWallet() {
             />
           </Tooltip>
 
-          <Tooltip
-            fontSize="xs"
-            label="Change account"
-            color="gold.200"
-            px={3}
-            py={2}
-          >
-            <IconButton
-              variant="ghost"
-              aria-label="Change account"
-              icon={<Icon as={IconUserCode} boxSize={5} />}
-              px={2}
-              boxSize={5}
-              onClick={() => handleConnectWallet(true)}
-            />
-          </Tooltip>
+          {isChangeAccountFeatureSupported(embeddedApp) && (
```

Comment:
Non-blocking comment

I think we don't need to create a separate function. We can simply use `isEmbed`.

Reply:
I think it depends on the embedded app. For example, we can support it in Ledger Live but not in Xverse.